_layouts/docs-default.html

```diff
@@ -18,13 +18,28 @@
       logTopSecretMessage("Don't worry, our site isn't collecting any data about you, or anything, really. Pinky promise.");
       logTopSecretMessage("You look like somebody who likes web programming and hacking. Why not join our dev team? Check out https://teachla.uclaacm.com/dev");
     </script>
-    <div class="container container-fill
+    
+    {% if page.title == 'The Teach LA Docs' %}
```

Comment:
I'm pretty sure that rather than manually check this, we could instead just specify the layout of the docs home ([`_docs/index.html`](https://github.com/uclaacm/teach-la-website/blob/docs_sidebar/_docs/index.html)) so that it doesn't inherit the default value we give it in our `_config.yml`. Saves the coder a little bit of time reading over the layout.